src/azure-cli/azure/cli/command_modules/rdbms/commands.py

```diff
@@ -297,3 +343,48 @@ def load_command_table(self, _):
         g.command('delete', 'delete', confirmation=True)
         g.show_command('show', 'get')
         g.command('list', 'list_by_server')
+
+    with self.command_group('mariadb server private-endpoint-connection',
+                            mariadb_private_endpoint_connections_sdk,
+                            client_factory=cf_mariadb_private_endpoint_connections_operations) as g:
+        g.custom_command('approve', 'approve_private_endpoint_connection',
+                         validator=validate_private_endpoint_connection_id)
+        g.custom_command('reject', 'reject_private_endpoint_connection',
+                         validator=validate_private_endpoint_connection_id)
+        g.command('delete', 'delete', validator=validate_private_endpoint_connection_id)
+        g.show_command('show', 'get', validator=validate_private_endpoint_connection_id)
+
+    with self.command_group('mariadb server private-link-resource',
+                            mariadb_private_link_resources_sdk,
+                            client_factory=cf_mariadb_private_link_resources_operations) as g:
+        g.show_command('list', 'list_by_server')
+
+    with self.command_group('mysql server private-endpoint-connection',
+                            mysql_private_endpoint_connections_sdk,
+                            client_factory=cf_mysql_private_endpoint_connections_operations) as g:
+        g.custom_command('approve', 'approve_private_endpoint_connection',
```

Comment:
It's a long running operation. Please support `--no-wait` and add `g.wait` command. You can search our command authoring.
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md by `(4) Supporting --no-wait` and `wait_command`

Reply:
the operation is very quick (1-3 seconds at max) for OSS databases. Hence avoiding adding --nowait and wait commands